src/pages/UserDetails/UserDetails.js

```diff
@@ -11,18 +11,49 @@ const containerStyles = css`
 	min-width: 420px;
 `;
 
+function getUrlParams(url) {
+	if (!url) {
+		return {};
+	}
+	const searchParams = new URLSearchParams(url);
```

Comment:
Are you using any package for `URLSearchParams`?
If not then use some package, it might not be present in some browsers.

Reply:
right adding the package